### PR TITLE
[TTAHUB-2042] Fix FEI root causes

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

A handful of grants in region six have invalid root causes set. This migration updates the table values to be the correct root cause.

## How to test

Run the migration all root causes should match the PDF on the ticket.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2042

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Review by thewatermethod — COMMENTED** [review]
Does this need to update the "GoalFieldResponses" table?

**Comment by AdamAdHocTeam** [comment]
> Does this need to update the "GoalFieldResponses" table?

Yea I think that's correct as the initial data is wrong.

File: src/migrations/20230908155325-fix-fei-root-causes.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                argfr.id
+                FROM "Grants" gr
+                JOIN "Goals" goal
+                    ON gr.id = goal."grantId"
+                JOIN "ActivityReportGoals" arg
+                    ON goal."id" = arg."goalId"
+                JOIN "ActivityReportGoalFieldResponses" argfr
+                    ON arg.id = argfr."activityReportGoalId"
+                WHERE "number" IN ('06CH011469', '06CH011503', '06CH011919'))
+                UPDATE "ActivityReportGoalFieldResponses"
```
Comment (thewatermethod) [inline src/migrations/20230908155325-fix-fei-root-causes.js:54]: When I queried the prod snapshot I have from a few weeks ago, none of these had "ActivityReportGoalFieldResponses" set at all. 

Maybe we should ask HHS if it makes sense to add the data to the AR's that have already been approved or just the goals?

Comment (AdamAdHocTeam) [reply src/migrations/20230908155325-fix-fei-root-causes.js:54]: Confirmed at stand up update both.

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**